workers/unity/Assets/Playground/Config/SystemConfig.cs

```diff
@@ -55,6 +57,8 @@ public static void AddGameLogicSystems(World world)
             CoreSystemHelper.AddSystems(world);
             TransformSynchronizationSystemHelper.AddSystems(world);
             PlayerLifecycleConfig.AddServerSystems(world);
+            GameObjectSystemHelper.AddSystems(world);
+            GameObjectCreationSystemHelper.AddSystems(world);
```

Comment:
Would it make sense to combine the two system helpers into one? You can't use any of the GameObject representation systems without using the gameobject creation systems anyway.

Reply:
The idea is that the 2nd is something you could write for yourself based on the exposed EntityGameObjectLinker. Could have two methods in the same static class though once I restructure a few things - at the moment the 2nd one needs to be in a separate assembly because it needs access to generated code, but this is probably changing based on some discussions yesterday